services/API-service/src/scripts/json/countries.json

```diff
@@ -440,7 +440,6 @@
         ],
         "eapLink": "https://510ibfsystem.blob.core.windows.net/about-trigger/MWI-flashfloods-about.pdf",
         "enableEarlyActions": false,
-        "enableStopTrigger": false,
```

Comment:
Was `enableStopTrigger` configured only for MWI flash-flood? This means all other countries and hazards were allowed to stop a forecasted trigger. Right?

Reply:
correct